doc/api/events.md

```diff
@@ -118,6 +124,8 @@ When a listener is registered using the `eventEmitter.on()` method, that
 listener will be invoked _every time_ the named event is emitted.
 
 ```js
+'use strict';
+
```

Comment:
This example both works without `use strict` and is partial (a continuation of the previous example), that means that this is not the correct place for `use strict`.